### ISSUE TITLE
blocklauncher still crashing - doesn't work

### ISSUE THREAD

**Issue by Creeperface01** [zhuowei/MCPELauncher#231]
Accer iconia a1-840FHD crashing
Acer liquid works

Both use Intel so repair it.

**Comment by zhuowei** [comment]
And they're running the same Minecraft PE APK and BlockLauncher version?

**Comment by Creeperface01** [comment]
I think no but i try it. And in version 0.10.5 you said "blocklauncher doesn't support Intel" but on acer liquid bl worked perfect! And on my tab a1-840FHD mods didn't work after firmware upgrade (added only split screen)

**Comment by zhuowei** [comment]
I don't have an Intel device myself, so I really don't know what's going on here. I know that there are two versions of MCPE that will run on an Intel device: the ARM version (for regular phones), and a special version of MCPE optimized for Intel Atom devices. If you downloaded MCPE from Google Play on an Intel device, you're getting the Intel optimized version; otherwise, you're probably getting the ARM version.

**Comment by Creeperface01** [comment]
THX a lot i have arm version :D so lets go to buy MCPE! and when i'll have Intel version of MCPE so bl won't work?

**Comment by zhuowei** [comment]
No, neither works properly right now - I've been mostly looking into compatibility with the Intel version though. Which version is installed on the Liquid?

**Comment by Creeperface01** [comment]
ARM
